dist/app.js

```diff
@@ -2,14 +2,13 @@ let app = angular.module('app', ['fs-widgets']);
 
 app.config( FsDataProvider => {
   FsDataProvider.customerId = 300033;
-  FsDataProvider.fromDate = '2017-07-01';
-  FsDataProvider.toDate = '2017-08-01';
-  FsDataProvider.dataUrl = '/data?type={{type}}&customer_id={{customerId}}&from_date={{fromDate}}&to_date={{toDate}}';
   FsDataProvider.baseUrl = 'http://localhost:3000';
+  FsDataProvider.dataUrl = '/data';
 });
 
-app.controller('MyCtrl', MyCtrl);
+app.controller('FsCtrl', FsCtrl);
 
-MyCtrl.$inject = ['$scope'];
-
-function MyCtrl($scope) { }
\ No newline at end of file
+function FsCtrl() {
+    this.fromDate = '2017-07-01';
+    this.toDate = '2017-08-01';
```

Comment:
If you want to have a timespan by default, you can set it here. Otherwise you will get the whole dataset.